traffic_ops/traffic_ops_golang/auth/tenancy.go

```diff
@@ -50,7 +53,7 @@ func GetUserTenantList(user CurrentUser, db *sqlx.DB) ([]Tenant, error) {
 	}
 	defer rows.Close()
 
-	Tenants := make([]Tenant, 0, 3)
+	Tenants := []Tenant{}
```

Comment:
I'll look closer tomorrow,  but this shouldn't be an exported name..  Can you change that?